### ISSUE TITLE
Octolapse disables Auto Bed Leveling

### ISSUE THREAD

**Issue by jalanjarosz** [FormerLurker/Octolapse#262]
This is a 2-part issue that I've encountered.

1) If I use my current Start G-Code, which extends outside the bounding box by 5mm, I get an Octolapse error stating that that.
2) If I adjust my Start G-Code, Octolapse disables Auto Bed Leveling.

Question: Is there a better location to move the ABL G-Code?

**Start G-Code:**
```
G21 ;metric values
G90 ;absolute positioning
M82 ;set extruder to absolute mode
M107 ;start with the fan off
G28 ;move X/Y/Z to min endstops
G29 ;auto level
M500; Save data to Eeprom
M501; recall data
M420 S1; implement mesh data in print
G1 Z15.0 F9000 ;move the platform down 15mm
G92 E0 ;zero the extruded length
G1 F200 E3 ;extrude 3mm of feed stock
G0 F7200.000000 X5.0 Y105.0 Z0.300
G92 E0
G1 F1800.000 X5.0 Y5.0 E5.29549
G1 X4.0 Y4.0 E10.63369
G1 X102.0 Y4.0 E16.0
G1 Z30.0 Y-5.0 E13.0
G1 E16
G92 E0 ;zero the extruded length again
G1 F9000
M117 Printing...
```

Here is the offending line that puts Octolapse outside the bounding box:
`G1 Z30.0 Y-5.0 E13.0`

Changing the Y value to within the bounding box (0 or positive value) or adjusting the Octolapse bounding box to include Y-5.0 will disable ABL.

Here is the ABL code:
Get the Mesh and save it to EEPROM:
```
G29 ;auto level
M500; Save data to Eeprom
```

This is the code that enables the use of the bed mesh during print:
```
M501; recall data
M420 S1; implement mesh data in print
```

Where should the M501 and M420 S1 be moved to?

**Comment by FormerLurker** [comment]
Hi!  What version of Octolapse are you using, 0.3.1 maybe?  I'm asking because this last ok like something I've already fixed.

**Comment by jalanjarosz** [comment]
Octolapse 0.3.1 is currently installed.

You gave me the latest RC build for multiple cameras, so I'll install that and let you know.

**Comment by jalanjarosz** [comment]
I'm happy to report that v0.3.4rc1 has not given me the out of bounds error, and I adjusted the Octolapse bounding box to include my Y-5.0 move AND ABL is working.

On a side note, I'm using a Logitech C930 camera, and the basic camera controls are working.  I've disabled auto-focus, set my focal length and zoomed in 130. (dual cam is next).

I think we can close this issue and I'll continue to run the RC code and provide any issues I find.

Thanks!
